ContactsMRC/ContactsMRC/ViewControllers/LSIContactsViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  LSIContactsViewController.h
+//  ContactsMRC
+//
+//  Created by Bhawnish Kumar on 6/17/20.
+//  Copyright © 2020 Bhawnish Kumar. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "LSIContactController.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LSIContactsViewController : UIViewController
+
+@property (nonatomic, assign) LSIContactController *contactController;
+@property (nonatomic, assign) LSIContact *contact;
```

Comment:
These should be retain, otherwise we can crash if the two objects disappear under our feet.

```objc
// Assign will not take ownership, so we want to use retain instead
// Pass by reference
@property (nonatomic, retain) LSIContactController *contactController;
@property (nonatomic, retain) LSIContact *contact;

```